src/EncompassRest/Contacts/BorrowerContacts.cs

```diff
@@ -0,0 +1,145 @@
+            GetBorrowerContactInternalAsync(contactId, cancellationToken, response => response.Content.ReadAsAsync<BorrowerContact>());
+
+        public Task<string> GetBorrowerContactRawAsync(string contactId) => GetBorrowerContactRawAsync(contactId, CancellationToken.None);
+        public Task<string> GetBorrowerContactRawAsync(string contactId, CancellationToken cancellationToken) =>
+            GetBorrowerContactInternalAsync(contactId, cancellationToken, response => response.Content.ReadAsStringAsync());
+
+        private async Task<T> GetBorrowerContactInternalAsync<T>(string contactId, CancellationToken cancellationToken, Func<HttpResponseMessage, Task<T>> func)
+        {
+            Preconditions.NotNullOrEmpty(contactId, nameof(contactId));
+
+            using (var response = await Client.HttpClient.GetAsync($"{s_apiPath}/{contactId}", cancellationToken).ConfigureAwait(false))
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    throw response.StatusCode == HttpStatusCode.NotFound ? await NotFoundException.CreateAsync($"{nameof(GetBorrowerContactAsync)}/{contactId}", response).ConfigureAwait(false) : await RestException.CreateAsync($"{nameof(GetBorrowerContactAsync)}", response).ConfigureAwait(false);
+                }
+                return await func(response).ConfigureAwait(false);
+            }
+        }
+        public Task<string> CreateBorrowerContactAsync(BorrowerContact contact) => CreateBorrowerContactAsync(contact, true, CancellationToken.None);
+        public Task<string> CreateBorrowerContactAsync(BorrowerContact contact, bool populate) => CreateBorrowerContactAsync(contact, populate, CancellationToken.None);
+        public Task<string> CreateBorrowerContactAsync(BorrowerContact contact, bool populate, CancellationToken cancellationToken)
+        {
+            Preconditions.NotNull(contact, nameof(contact));
+
+            return CreateBorrowerContactInternalAsync(JsonStreamContent.Create(contact), populate ? new QueryParameters(new QueryParameter("view", "entity")).ToString() : null, cancellationToken, async response =>
+            {
+                if (populate)
+                {
+                    await response.Content.PopulateAsync(contact).ConfigureAwait(false);
+                }
+                contact.Dirty = false;
+                return Path.GetFileName(response.Headers.Location.OriginalString);
+            });
+        }
+
+        public Task<string> CreateBorrowerContactRawAsync(string contact) => CreateBorrowerContactRawAsync(contact, "", CancellationToken.None);
+        public Task<string> CreateBorrowerContactRawAsync(string contact, CancellationToken cancellationToken) => CreateBorrowerContactRawAsync(contact, null, cancellationToken);
+        public Task<string> CreateBorrowerContactRawAsync(string contact, string queryString) => CreateBorrowerContactRawAsync(contact, queryString, CancellationToken.None);
```

Comment:
Make method private as this is not currently supported by the Encompass API.

Reply:
leaving this for now, because there is no impact on the API.